src/Websites/Websites/Cmdlets/Certificates/RemoveAzureWebAppCertificate.cs

```diff
@@ -0,0 +1,94 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.WebApps.Models;
+using Microsoft.Azure.Commands.WebApps.Utilities;
+using Microsoft.Azure.Management.WebSites.Models;
+using System.Linq;
+using System.Management.Automation;
+using System.Net;
+
+namespace Microsoft.Azure.Commands.WebApps.Cmdlets.Certificates
+{
+    /// <summary>
+    /// This commandlet will let you delete a managed certificate
+    /// </summary>
+    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "WebAppCertificate", SupportsShouldProcess = true), OutputType(typeof(void))]
+    public class RemoveAzureWebAppCertificate : WebAppBaseClientCmdLet
+    {
+        const string ParameterSet1Name = "S1";
+
+        [Parameter(ParameterSetName = ParameterSet1Name, Position = 0, Mandatory = true, HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSet1Name, Position = 1, Mandatory = true, HelpMessage = "The name of the web app.")]
+        [ResourceNameCompleter("Microsoft.Web/sites", "ResourceGroupName")]
+        [ValidateNotNullOrEmpty]
+        public string WebAppName { get; set; }
```

Comment:
We don't need WebappName for delete. Please remove.

Reply:
we need WebAppName sisira. Because if SSL binding is added to the webapp to the deleting certificate then first we need to remove the `sslstate` then remove the cert. otherwise it will throw an exception.

Reply:
Like we discussed let the API do the right thing. don't delete the SSL binding for the user.

Reply:
Sure sisira. Done as per our discussion.